langlib/lang.xml/src/main/java/org/ballerinalang/langlib/xml/GetName.java

```diff
@@ -0,0 +1,57 @@
+/*
+ *   Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  WSO2 Inc. licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.ballerinalang.langlib.xml;
+
+import org.ballerinalang.jvm.Strand;
+import org.ballerinalang.jvm.util.exceptions.BLangExceptionHelper;
+import org.ballerinalang.jvm.util.exceptions.RuntimeErrors;
+import org.ballerinalang.jvm.values.XMLValue;
+import org.ballerinalang.model.types.TypeKind;
+import org.ballerinalang.natives.annotations.Argument;
+import org.ballerinalang.natives.annotations.BallerinaFunction;
+import org.ballerinalang.natives.annotations.ReturnType;
+
+/**
+ * Returns the string giving the expanded name of provided xml element.
+ *
+ * @since 1.0
+ */
+@BallerinaFunction(
+        orgName = "ballerina", packageName = "lang.xml",
+        functionName = "getName",
+        args = {@Argument(name = "xmlValue", type = TypeKind.XML)},
+        returnType = {@ReturnType(type = TypeKind.STRING)},
+        isPublic = true
+)
+public class GetName {
+
+    private static final String OPERATION = "get element name in xml";
+
+    public static String getName(Strand strand, XMLValue<?> xmlVal) {
+        if (!IsElement.isElement(strand, xmlVal)) {
+            throw BLangExceptionHelper.getRuntimeException(RuntimeErrors.XML_FUNC_TYPE_ERROR, "getName", "element");
+        }
+        try {
+            return xmlVal.getElementName();
+        } catch (Throwable e) {
+            BLangExceptionHelper.handleXMLException(OPERATION, e);
+        }
+
+        return null;
```

Comment:
Shouldn't be returning `null` right? What's the appropriate value here? Empty string?

Reply:
`BLangExceptionHelper.handleXMLException(OPERATION, e)` always throws so control never reach this point.
I would like to put a `assert false` here but since it's not being used anywhere else just went with a null.